moxin-frontend/src/my_models/downloaded_files_table.rs

```diff
@@ -544,59 +540,45 @@ pub enum ButtonType {
 #[derive(Live, LiveHook, Widget)]
 pub struct ActionButton {
     #[deref]
-    view: View,
+    deref: CButton,
```

Comment:
This is a NIT, but we are using the name of the widget we deref everywhere else, so it itches me a bit.